styles/navigation.scss

```diff
@@ -17,8 +17,10 @@
   }
   &__scroll_down {
     top: -73px;
+    z-index: 5;
     .nav__logo-wrap {
       top: 70px;
+      z-index: 5;
```

Comment:
zmieniłeś z-idex całego wrpa, potrzebujemy tylko samo logo. Teraz 2 pierwsze itemy w nav za przez to przykryte